### PR TITLE
Minify css output

### PR DESCRIPTION
**Pull request by stephane-vanraes**

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -1,20 +1,17 @@
...
+    /* Svelte command takes an optional third argument (options) with fields:
+        legacy (default false): compiles to es5
+        plugins: extra plugins to run after default ones
+    */
+    Svelte('./src/index.js', dest + 'index.bundle.js'),
+    Svelte('./src/index.js', dest + 'index.bundle.es5.js', { legacy: true }),
+    /* Sass command takes an optional thrid arguments (options) with fields:
+        plugins: extra plugins to run after default ones
+    */
+    Sass('./src/theme.aaa.scss', dest + 'aaa.bundle.css'),
```
Comment (furstenberg) [inline rollup.config.js]: Should `dest` be `destination` perhaps?

Comment (stephane-vanraes) [reply rollup.config.js]: Yes, my bad :)

File: rollup.config.js [hunk 2]
```diff
@@ -1,20 +1,17 @@
...
+const destination = './build/';
+
+export default [
+    /* Svelte command takes an optional third argument (options) with fields:
+        legacy (default false): compiles to es5
+        plugins: extra plugins to run after default ones
+    */
+    Svelte('./src/index.js', destination + 'index.bundle.js'),
+    Svelte('./src/index.js', destination + 'index.bundle.es5.js', { legacy: true }),
+    /* Sass command takes an optional thrid arguments (options) with fields:
```
Comment (furstenberg) [inline rollup.config.js]: typo. Should be `third`.

**Review by furstenberg — APPROVED** [review]
There's a typo in a comment, but this looks good to merge.

File: rollup.config.js (hunk 2 above)
Comment (stephane-vanraes) [reply rollup.config.js]: thanks, fixed it

**Pull request merged**